jsonnet/components/thanos-querier.libsonnet

```
@@ -144,7 +149,7 @@ function(params)
               },
             },
             resourceAttributes: {
-              apiVersion: 'metrics.k8s.io/v1beta1',
+              apiVersion: 'v1',
```

IIUC the API group needs to metrics.k8s.io and the API version is [v1beta1](https://github.com/kubernetes/metrics/tree/master/pkg/apis/metrics/v1beta1)
```suggestion
              apiVersion: 'v1beta1',
              apiGroup: 'metrics.k8s.io',
```

---

I set the `apiVersion` to `v1` considering the `resource` (in the line just beneath this) is set to `pod`s, which is available in `core/v1`, am I missing something?

---

I went through the KRP and `SubjectAccessReview` documentation (and types) figuring this out because for some reason I kept associating `pods` to `core/v1`. 🤦🏼

---

Now that I've gained common sense, do you think it'd make sense to add a `sub resource: metrics` field in here as well?

---

Is this actually resolved? It's still `apiVersion: 'v1'` and missing the `apiGroup` value..?